src/index.js

```diff
@@ -1,32 +1,39 @@
-var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+//Esta linea detenia la ejecución del scrip dado que no se cargo el modulo requireJS
+//var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 
-var API = 'https://rickandmortyapi.com/api/character/';
+var API = "https://rickandmortyapi.com/api/character/";
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```